user-docs/transaction-functions.md

```diff
@@ -0,0 +1,121 @@
+# Transaction Functions available in `algob`
+
+## `getSuggestedParams`
+
+This function performs the task of fetching the common transaction params from the connected network blockchain. Private chains might throw some error while executing this function, please make sure that the node progresses in that case. 
+
+This function accepts:
++ `algoc1`: an instance(object) of `Algodv2` class of `algosdk`, which is to be used to fetch the transaction params from the connected network. 
+
+This function returns:
++ a promise on resolving which we get the desired transaction params.
```

Comment:
what do you mean desired ?

Reply:
I'll change this to suggested params.

Reply:
this line is not correct.
Anyway - let's remove this style of documentation and use proper API docs.

Reply:
In the API  this almost doesn't require any documentation. We have types and everything is in context.